float32 y
float32 z
float32 phi
float32 ydot
float32 zdot
float32 phidot

